### ISSUE TITLE
Installation broken on brand-new Digital Ocean box

### ISSUE THREAD

**Issue by miracle2k** [mail-in-a-box/mailinabox#1081]
This happens to be every time:

1. Set up a fresh Digital Ocean box with 14.04.
2. Do apt-get update && apt-get upgrade.
3. Run curl -s https://mailinabox.email/setup.sh | sudo bash

It fails here:

```
FAILED: pip3 install --upgrade rtyaml email_validator>=1.0.0 free_tls_certificates>=0.1.3 idna>=2.0.0 cryptography>=1.0.2 boto psutil
.....
.....
ImportError: No module named 'packaging'
```
Full error here: https://gist.github.com/miracle2k/3f84f408db3a3143a01ad27c2645554c

Afterwards, pip3 is completely broken:

```
root@mail1:~# pip3
Traceback (most recent call last):
  File "/usr/bin/pip3", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python3.4/dist-packages/pkg_resources/__init__.py", line 70, in <module>
    import packaging.version
ImportError: No module named 'packaging'
```

**Comment by miracle2k** [comment]
It seems to be related to `cryptography` pulling in a setuptools update.

I can make it work by manually upgrading setuptools first, before installing mailinabox:

`pip3 install --upgrade setuptools`

**Comment by cromulus** [comment]
Perhaps make a pull request with those changes?

**Comment by AshFrancis** [comment]
miracle2k, could you let me know the steps to fix my installation if possible. I upgraded from 0.20 to 0.21b and pip3 is now broken as is the admin status checks.

**Comment by cromulus** [comment]
I have replicated this. 
Will try to put together a PR as this is currently a blocker for me adding new hosts.

**Comment by cromulus** [comment]
Looks like it's an upstream issue with pip: pypa/pip#4253
I've tried the manual installation of setuptools with `pip3 install --upgrade setuptools` and I'm still getting the error.

I'm not exactly sure how to proceed.

**Comment by miracle2k** [comment]
@AshFrancis I wasn't able to fix it, I just tried a fresh VM.

**Comment by akrawchyk** [comment]
I ran into this as well and was able to workaround the issue by upgrading `pip3` using `easy_install`:

```
sudo easy_install -U pip
```

**Comment by AshFrancis** [comment]
Thank you both, this fix from the discourse seems to have worked:
https://discourse.mailinabox.email/t/unable-to-run-mailinabox-command/1840/14

**Comment by cromulus** [comment]
duplicate of #1080

**Comment by JoshData** [comment]
Fixed in v0.21c posted this morning.
